cmd/frontend/internal/httpapi/auth.go

```diff
@@ -70,8 +71,14 @@ func AccessTokenAuthMiddleware(db database.DB, next http.Handler) http.Handler {
 			}
 			subjectUserID, err := db.AccessTokens().Lookup(r.Context(), token, requiredScope)
 			if err != nil {
-				log15.Error("Invalid access token.", "token", token, "err", err)
-				http.Error(w, "Invalid access token.", http.StatusUnauthorized)
+				if err == database.ErrAccessTokenNotFound || errors.HasType(err, database.InvalidTokenError{}) {
+					log15.Error("Invalid access token.", "token", token, "err", err)
```

Comment:
```suggestion
					log15.Error("AccessTokenAuthMiddleware.invalidAccessToken", "token", token, "error", err)
```

Since you've touched this line, we could take this chance to improve the logging (because we don't print file line, it is always confusing to get where the error is coming from).